functions/grapher/csv/[slug].ts

```diff
@@ -0,0 +1,51 @@
+import { fetchCsvForGrapher } from "../../_common/grapherRenderer.js"
+import { IRequestStrict, Router, error } from "itty-router"
+
+export interface Env {
+    ASSETS: {
+        fetch: typeof fetch
+    }
+    url: URL
+}
+
+const router = Router<IRequestStrict, [URL, Env, ExecutionContext]>()
+router
+    .get(
+        "/grapher/csv/:slug",
+        async ({ params: { slug } }, { searchParams: _ }, env) =>
+            fetchCsvForGrapher(slug, env)
+    )
+    .all("*", () => error(404, "Route not defined"))
```

Comment:
This suggests to me that either the filename doesn't matter, or the routing (nearly) doesn't matter.